content/codespaces/prebuilding-your-codespaces/managing-prebuilds.md

```diff
@@ -52,13 +52,13 @@ This displays the workflow run history for prebuilds for the associated branch.
 ## Editing a prebuild configuration
 
 1. On the {% data variables.product.prodname_codespaces %} page of your repository settings, click the ellipsis to the right of the prebuild configuration you want to edit.
-1. In the dropdown menu, click **Edit**.
+2. In the dropdown menu, click **Edit**.
```

Comment:
We intentionally use 1. for all numbers in an ordered list. The site renders the numbers correctly, and we don't need to update them all if a step is added or removed. This change is causing the linter test to fail.

```suggestion
1. In the dropdown menu, click **Edit**.
```